style.css

```diff
@@ -0,0 +1,123 @@
+* {
+  margin: 0;
+  padding: 0; 
+  box-sizing: border-box;
+}
+
+.meditation-app {
+  height: 100vh;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.container__time, .container__sounds, .container__player {
+  height: 80%;
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.container__player {
+  position: relative;
+}
+
+.track-outline, .moving-outline {
+  position: absolute;
+  height: 50%;
+  transform: rotate(-90deg);
+}
+
+.time-display {
+  position: absolute;
+  bottom: 10%;
+  font-size: 5rem;
+  color: #ffffff;
+}
+
+video {
+  position: fixed;
+  top: 0%;
+  left: 0%;
+  height: 100vh;
+  z-index: -1;
+}
+
+button {
+  color: #ffffff;
+  width: 30%;
+  height: 10%;
+  background: none;
+  border: 2px solid #ffffff;
+  border-radius: 5px;
+  cursor: pointer;
+  font-size: 1.3rem;
+  transition: all 0.5s ease ;
+}
+
+.time-select:hover {
+  color: black;
+  background: #ffffff;
+}
+
+.container__sounds button {
+  border: none;
+  height: 120px;
+  width: 120px;
+  border-radius: 50%;
+}
+
+.container__sounds button img {
+  height: 70%;
+}
+
+.play { 
+  z-index: 1;
+}
+
+@media (max-width: 769px) {
+  .track-outline, .moving-outline {
+    height: 35%;
+  }
+  button {
+    width: 53%;
+    height: 8%;
+  }
+}
+
+@media (max-width: 411px) {
+  button {
+    margin: 10px;
+    width: 28%;
+    height: 18%;
+  }
+  .meditation-app {
+    display: flex;
+    flex-direction: column;
+  }
+  .container__time, .container__sounds {
+    flex-direction: row;
+  }
+  .container__player {
+    position: absolute;
+  }
+  .time-display {
+    bottom: 16%;
+  }
+  .container__sounds {
+    position: absolute;
+    top: 47%;
+  }
+  .container__time {
+    position: absolute;
+    top: -21%;
+  }
+}
+
+@media (max-width: 375px) {
+  .time-display {
+    font-size: 4rem;
+  }
+}
```

Comment:
I have such a view on the desktop: the background video doesn't fit the screen container.
![image](https://user-images.githubusercontent.com/45519864/115101652-a76aec80-9f4e-11eb-8f64-acf9265bf97d.png)

And on some screens the button labels don't fit the button container.
![image](https://user-images.githubusercontent.com/45519864/115101676-c9fd0580-9f4e-11eb-899d-3291c44e2d86.png)

Reply:
DONE!